### PR TITLE
Added email plugin instructions

### LINKED ISSUE #6438: Access Request reviewers should receive a notification email
### What
As of Teleport 6.1 you can specify reviewers when creating an access request. Teleport should be able to email these reviewers to alert them about the access request.

### How
Add an area to specify SMTP settings?

### PR DESCRIPTION
**Pull request by stevenGravy**
Closes #6438

### CODE REVIEW

**Comment by russjones** [comment]
@stevenGravy Please apply changes and update thie PR.

**Comment by ptgott** [comment]
Hi @stevenGravy, just checking if we still want to add this guide to the docs. If so (it seems really useful to me), would you mind rebasing and I'll give this a look? Thanks!

**Comment by stevenGravy** [comment]
> Hi @stevenGravy, just checking if we still want to add this guide to the docs. If so (it seems really useful to me), would you mind rebasing and I'll give this a look? Thanks!

Yes, will do.

File: docs/pages/enterprise/workflow/ssh-approval-email.mdx [hunk 1]
```diff
@@ -0,0 +1,327 @@
+---
+title: Teleport integration with Email
+description: This guide explains how to setup a Email plugin for Teleport for privilege elevation approval notification.
+h1: Teleport Email Plugin Setup
+---
+
+This guide will talk through how to setup Teleport with Email notification for Access Requests.
```
Comment (ptgott) [inline docs/pages/enterprise/workflow/ssh-approval-email.mdx:7]:
```suggestion
This guide will explain how to set up Teleport with email notifications for Access Requests.
```

File: docs/pages/enterprise/workflow/ssh-approval-email.mdx [hunk 2]
```diff
@@ -0,0 +1,327 @@
+---
+title: Teleport integration with Email
```
Comment (ptgott) [inline docs/pages/enterprise/workflow/ssh-approval-email.mdx:2]:
```suggestion
title: Teleport Access Requests with Email
```

File: docs/pages/enterprise/workflow/ssh-approval-email.mdx [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+description: This guide explains how to setup a Email plugin for Teleport for privilege elevation approval notification.
```
Comment (ptgott) [inline docs/pages/enterprise/workflow/ssh-approval-email.mdx:3]:
```suggestion
description: This guide explains how to set up the Teleport email plugin to notify users when another user requests elevated privileges.
```

File: docs/pages/enterprise/workflow/ssh-approval-email.mdx [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+h1: Teleport Email Plugin Setup
```
Comment (ptgott) [inline docs/pages/enterprise/workflow/ssh-approval-email.mdx:4]:
```suggestion
```
I think we can stick with the title here.

File: docs/pages/enterprise/workflow/ssh-approval-email.mdx [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+---
+
+This guide will talk through how to setup Teleport with Email notification for Access Requests.
+
+
+## Setup
```
Comment (ptgott) [inline docs/pages/enterprise/workflow/ssh-approval-email.mdx:10]: We should use the H2 format that other guides use: a `## Prerequisites` section followed by H2 headings in the format `## Step n/d. Do something`. This way, readers can tell how far they are into the guide.

**Review by ptgott — COMMENTED** [review]
`ssh-approval-email.mdx` fails to render because of a `<Tabs>` tag balancing issue. 

I also left a couple of small suggestions and a structural comment to start. Once these are incorporated (and the page renders), I'll give the whole guide a more thorough look.

**Comment by ptgott** [comment]
I am going to base a new PR off this branch to edit this guide and shepherd it to completion this week!

**Pull request closed without merging**